### ISSUE TITLE
Leaflet Map not displaying on startup page

### ISSUE THREAD

**Issue by cchasson** [JeroenDeDauw/Maps#655]
The problem is simple. SMW is working here with leaflet map: http://www.gjwiki.fr/index.php?title=Mod%C3%A8le:Accueil/RP

The same mediawiki embedded in the startup page is not working see: http://www.gjwiki.fr/index.php?title=Accueil (the map is not loading, in french "chargement de la page")

I have tried with googlemaps, and i have tried to directly had the code in the page, same result.

Thanks for your help and to take time to see the issue.

Charles

**Comment by JeroenDeDauw** [comment]
![image](https://user-images.githubusercontent.com/146040/111031993-8b81a180-840a-11eb-8d24-16ef2d2f26af.png)

**Comment by cchasson** [comment]
Yes it is ok in the template page, but in the main page, you can see that it is not working ?

**Comment by JeroenDeDauw** [comment]
My guess is that something else on your main page is breaking the map. There is no error in the browser console though, so hard to tell.

**Comment by cchasson** [comment]
I think the same, i will first update mediawiki to the last build and i will advise. Thanks for your help.
